pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -40,9 +40,9 @@ type fatalError string
 // transientError is an error which should always be retried (indefinitely).
 type transientError string
 
-func (e fatalError) Error() string { return "fatal error handling deploymentConfig: " + string(e) }
+func (e fatalError) Error() string { return "fatal error handling deployment config: " + string(e) }
```

Comment:
I dont thing this is what the PR is about, if it was I would fix it also on other places. Let us focus on the PR goal :)